abgc_derive/src/lib.rs

```diff
@@ -1,7 +1,30 @@
-#[cfg(test)]
-mod tests {
-    #[test]
-    fn it_works() {
-        assert_eq!(2 + 2, 4);
-    }
+// Copyright (c) 2019 King's College London created by the Software Development Team
```

Comment:
There's either some punctuation missing or a misplaced newline around the first line of the copyright notice here.

FWIW, in yk we have:
```
// Copyright 2019 King's College London.
// Created by the Software Development Team <http://soft-dev.org/>.
...
```

Reply:
That's the same license formatting as yksom. I agree that it might be better reformatted, but I think maybe that can/should be a separate PR?

Reply:
It's up to you where we adjust it, but as it stands it sounds like our team created King's College London.

My bad for not spotting this in yksom.